File: CS/MeffedNavigationService.Infrastructure/Contract/IModuleMetadata.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MeffedNavigationService.Infrastructure.Enums;

namespace MeffedNavigationService.Infrastructure.Contract {
    public interface IModuleMetadata {
        ModuleTypes ModuleType { get; }
    }
}

File: VB/MeffedNavigationService.Infrastructure/Contract/IModuleMetadata.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports MeffedNavigationService.Infrastructure.Enums

Namespace MeffedNavigationService.Infrastructure.Contract
	Public Interface IModuleMetadata
		ReadOnly Property ModuleType() As ModuleTypes
	End Interface
End Namespace
